resources/authgear/templates/en/web/authflowv2/wechat.html

```diff
@@ -0,0 +1,78 @@
+{{ template "authflowv2/__page_frame.html" . }}
+
+{{ define "page-content" }}
+{{ $unknown_error_message := "" }}
+{{ if $.HasUnknownError }}
+  {{ $unknown_error_message = (include "authflowv2/__error.html" .) }}
+{{ end }}
+
+{{ if $.IsNativePlatform }}
+{{ if $.WechatRedirectURI }}
+
+<div class="screen-icon-layout flex-1-0-auto">
```

Comment:
I think we should not use `screen-icon-layout` if there isn't a srceen icon.

Reply:
Separated class name for this layout